recipes/osqp/all/conanfile.py

```diff
@@ -18,12 +18,29 @@ class OsqpConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "printing": [True, False],
+        "profiling": [True, False],
+        "ctrlc": [True, False],
+        "dfloat": [True, False],
+        "dlong": [True, False],
+        "coverage": [True, False],
+        "mkl_paradisio": [True, False]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "printing": True,
+        "profiling": True,
+        "ctrlc": True,
+        "dfloat": False,
+        "dlong": True,
+        "coverage": False,
+        "mkl_paradisio": True
     }
```

Comment:
An `options_description = {...}` field would be nice, since the meaning of the options isn't super obvious.